src/main/java/pvs/app/service/ProjectService.java

```diff
@@ -36,6 +39,13 @@ public void create(CreateProjectDTO projectDTO) throws IOException {
             addGithubRepo(addGithubRepositoryDTO);
         }
 
+        if (!projectDTO.getGitlabRepositoryURL().equals("")) {
```

Comment:
Suggest add `trim()` to prevent some unexpected inputs.
```suggestion
        if (!projectDTO.getGitlabRepositoryURL().trim().equals("")) {
```